internal/artifacts/module_arch_test.go

```diff
@@ -445,17 +471,17 @@ module-types:
 			When("build parameters has timeout", func() {
 				It("succeeds when timeout is not exceeded", func() {
 					ep := dir.Loc{SourcePath: getTestPath("mta"), TargetPath: getResultPath(), MtaFilename: "mta_with_timeout.yaml"}
-					Ω(buildModule(&ep, &ep, &ep, "m2", "cf")).Should(Succeed())
-					Ω(ep.GetTargetModuleZipPath("m2")).Should(BeAnExistingFile())
+					Ω(buildModule(&ep, &ep, "m2", "cf")).Should(Succeed())
+					Ω(getTestPath("result", ".mta_mta_build_tmp", "m2", "data.zip")).Should(BeAnExistingFile())
```

Comment:
Can't we use `getFullPathInTmpFolder` instead of creating this path everywhere? (If it doesn't exist in this package copy it...)

Reply:
done